app/src/main/java/xyz/jhughes/laundry/activities/LocationActivity.java

```diff
@@ -37,18 +40,27 @@
     @Bind(R.id.location_activity_toolbar) Toolbar toolbar;
     @Bind(R.id.progressBar) ProgressBar mLoadingProgressBar;
 
-
     private LocationAdapter adapter;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
```

Comment:
See, I was having a ton of trouble with this particular feature since the lifecycle says that the exact same methods are called when the user does a cold open and when they navigate back following the event of the phone running low on memory, so unless you have a suggestion, i couldn't find a different good way to check that